code/edgebridge/src/main/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridge.java

```diff
@@ -46,18 +46,20 @@ public static String extensionVersion() {
 	 * Registers the Edge Bridge extension with the Mobile Core.
 	 * This method should be called before calling {@link MobileCore#start(AdobeCallback)}.
 	 */
```

Comment:
Can you please add a comment that redirects to MobileCore.registerExtensions(List, AdobeCallback) usage?